src/backend/src/generated/telemetry/telemetry.pb.h

```diff
@@ -384,6 +384,12 @@ enum FlightMode : int {
   LAND = 6,
   OFFBOARD = 7,
   FOLLOW_ME = 8,
+  MANUAL = 9,
```

Comment:
Did you edit that manually, or did you run [generate_from_protos.sh](https://github.com/mavlink/MAVSDK/blob/develop/tools/generate_from_protos.sh)?

The `generated/` files should come from that auto-generation script :blush:.

Reply:
Thx, yes you are right I edited manual.
But when I call script I'm getting following error:

```
./tools/generate_from_protos.sh: 5: ./tools/generate_from_protos.sh: Bad substitution
-------------------------------
 Error
-------------------------------
'protoc' or 'grpc_cpp_plugin' not found

Those are expected to be built for the host system in '/home/hanco/Asli/MAVSDK/MAVSDK/../build/default/third_party'!

You may want to run the CMake configure step first:

    $ cmake -DBUILD_BACKEND=ON -Bbuild/default -S.

```

But I already have these files since I've used cmake config from this [Building the Backend](https://mavsdk.mavlink.io/develop/en/contributing/build.html#sdk_local_install)

> Building the Backend
> The MAVSDK programming-language-specific libraries (e.g. Swift, Python) share a common backend, which may optionally be built as part of the C++ library.
> 
> The cmake configuration step additionally depends on the -DBUILD_BACKEND=ON option. Otherwise the build is exactly the same as usual.
> 
> When building the backend, we usually like to link all the dependencies statically, and therefore we set -DBUILD_SHARED_LIBS=OFF (or don't specify it, because the default is OFF).
> 
> Ubuntu
> To build the backend on Ubuntu:
> 
> Setup the C++ Library on Linux
> Navigate into the SDK directory and build the project
> cd MAVSDK
> cmake -DBUILD_BACKEND=ON -Bbuild/default -H.
> cmake --build build/default

And also FYI
```
hanco@CRT-boi:~$ protoc --version
libprotoc 3.0.0
```

Reply:
Can you check if you have a `protoc` or `grpc_cpp_plugin` in `/home/hanco/Asli/MAVSDK/MAVSDK/../build/default/third_party`? E.g.:

```
find /home/hanco/Asli/MAVSDK/MAVSDK/../build/default/third_party | grep protoc
```

Otherwise I'll try to do it myself ASAP :+1:.

Reply:
For grpc_cpp_plugin:

```
hanco@CRT-boi:~$ find /home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party | grep grpc_cpp
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/install/bin/grpc_cpp_plugin
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/tools/internal_ci/linux/sanitizer/pull_request/grpc_cpp_tsan.cfg
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/tools/internal_ci/linux/sanitizer/pull_request/grpc_cpp_asan.cfg
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/tools/internal_ci/linux/sanitizer/grpc_cpp_tsan.cfg
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/tools/internal_ci/linux/sanitizer/grpc_cpp_asan.cfg
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc-build/CMakeFiles/grpc_cpp_plugin.dir
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc-build/CMakeFiles/grpc_cpp_plugin.dir/CXX.includecache
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc-build/CMakeFiles/grpc_cpp_plugin.dir/flags.make
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc-build/CMakeFiles/grpc_cpp_plugin.dir/cmake_clean.cmake
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc-build/CMakeFiles/grpc_cpp_plugin.dir/src
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc-build/CMakeFiles/grpc_cpp_plugin.dir/src/compiler
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc-build/CMakeFiles/grpc_cpp_plugin.dir/src/compiler/cpp_plugin.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc-build/CMakeFiles/grpc_cpp_plugin.dir/progress.make
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc-build/CMakeFiles/grpc_cpp_plugin.dir/depend.internal
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc-build/CMakeFiles/grpc_cpp_plugin.dir/DependInfo.cmake
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc-build/CMakeFiles/grpc_cpp_plugin.dir/depend.make
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc-build/CMakeFiles/grpc_cpp_plugin.dir/link.txt
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc-build/CMakeFiles/grpc_cpp_plugin.dir/build.make
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc-build/grpc_cpp_plugin
```

For protoc:

```
hanco@CRT-boi:~$ find /home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party | grep protoc
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/libressl/libressl/src/libressl/man/tls_config_set_protocols.3
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/libressl/libressl/src/libressl/man/SSL_SESSION_get_protocol_version.3
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/install/bin/protoc
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/install/bin/protoc-3.9.1.0
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/install/lib/libprotoc.a
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/install/share/man/man3/tls_config_set_protocols.3
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/install/share/man/man3/SSL_SESSION_get_protocol_version.3
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/libprotoc.a
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/CXX.includecache
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/flags.make
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/cmake_clean.cmake
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/home
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/home/hanco
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/home/hanco/Asli
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/home/hanco/Asli/MAVSDK
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/home/hanco/Asli/MAVSDK/MAVSDK
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/main.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/progress.make
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/depend.internal
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/DependInfo.cmake
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/depend.make
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/link.txt
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/build.make
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/CXX.includecache
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/flags.make
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/cmake_clean.cmake
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/cmake_clean_target.cmake
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/ruby
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/ruby/ruby_generator.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/plugin.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/js
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/js/js_generator.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/js/well_known_types_embed.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/subprocess.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/cpp
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/cpp/cpp_map_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/cpp/cpp_padding_optimizer.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/cpp/cpp_message.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/cpp/cpp_message_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/cpp/cpp_primitive_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/cpp/cpp_generator.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/cpp/cpp_extension.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/cpp/cpp_enum.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/cpp/cpp_file.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/cpp/cpp_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/cpp/cpp_service.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/cpp/cpp_enum_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/cpp/cpp_string_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/cpp/cpp_helpers.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/php
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/php/php_generator.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/command_line_interface.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/objectivec
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/objectivec/objectivec_message_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/objectivec/objectivec_generator.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/objectivec/objectivec_file.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/objectivec/objectivec_enum.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/objectivec/objectivec_enum_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/objectivec/objectivec_extension.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/objectivec/objectivec_primitive_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/objectivec/objectivec_map_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/objectivec/objectivec_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/objectivec/objectivec_oneof.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/objectivec/objectivec_message.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/objectivec/objectivec_helpers.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/csharp
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/csharp/csharp_enum_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/csharp/csharp_repeated_message_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/csharp/csharp_repeated_primitive_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/csharp/csharp_primitive_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/csharp/csharp_source_generator_base.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/csharp/csharp_generator.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/csharp/csharp_enum.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/csharp/csharp_wrapper_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/csharp/csharp_reflection_class.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/csharp/csharp_doc_comment.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/csharp/csharp_message_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/csharp/csharp_repeated_enum_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/csharp/csharp_map_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/csharp/csharp_field_base.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/csharp/csharp_message.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/csharp/csharp_helpers.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/code_generator.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/zip_writer.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/python
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/python/python_generator.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/plugin.pb.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_enum_field_lite.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_file.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_string_field_lite.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_name_resolver.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_message.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_map_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_service.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_context.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_extension_lite.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_doc_comment.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_map_field_lite.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_enum.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_generator.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_primitive_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_primitive_field_lite.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_message_field_lite.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_extension.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_message_lite.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_message_builder_lite.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_message_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_shared_code_generator.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_message_builder.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_helpers.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_generator_factory.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_enum_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_string_field.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google/protobuf/compiler/java/java_enum_lite.cc.o
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/progress.make
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/depend.internal
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/DependInfo.cmake
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/depend.make
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/link.txt
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/libprotoc.dir/build.make
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/protoc
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/protoc-3.9.1.0
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/protoc-artifacts
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/protoc-artifacts/Dockerfile
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/protoc-artifacts/scl-enable-devtoolset.sh
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/protoc-artifacts/build-zip.sh
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/protoc-artifacts/README.md
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/protoc-artifacts/build-protoc.sh
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/protoc-artifacts/pom.xml
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/libprotoc.map
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/cmake/libprotoc.cmake
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/cmake/protoc.cmake
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/benchmarks/util/protoc-gen-proto2_to_proto3.cc
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/benchmarks/util/protoc-gen-gogoproto.cc
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/kokoro/release/protoc
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/kokoro/release/protoc/windows
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/kokoro/release/protoc/windows/release.cfg
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/kokoro/release/protoc/windows/build.bat
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/kokoro/release/protoc/linux
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/kokoro/release/protoc/linux/release.cfg
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/kokoro/release/protoc/linux/build.sh
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/kokoro/release/protoc/macos
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/kokoro/release/protoc/macos/release.cfg
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/kokoro/release/protoc/macos/build.sh
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/curl/curl/src/curl/docs/cmdline-opts/haproxy-protocol.d
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/info
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/info/exclude
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/description
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/config
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/HEAD
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/refs
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/refs/heads
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/refs/heads/master
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/refs/remotes
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/refs/remotes/origin
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/refs/remotes/origin/HEAD
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/refs/tags
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/index
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/logs
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/logs/HEAD
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/logs/refs
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/logs/refs/heads
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/logs/refs/heads/master
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/logs/refs/remotes
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/logs/refs/remotes/origin
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/logs/refs/remotes/origin/HEAD
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/objects
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/objects/info
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/objects/pack
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/objects/pack/pack-cd84226ebcf3cf156bb7758611a5bea36fee3ca3.pack
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/objects/pack/pack-cd84226ebcf3cf156bb7758611a5bea36fee3ca3.idx
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/hooks
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/hooks/post-update.sample
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/hooks/pre-push.sample
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/hooks/applypatch-msg.sample
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/hooks/fsmonitor-watchman.sample
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/hooks/pre-applypatch.sample
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/hooks/update.sample
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/hooks/prepare-commit-msg.sample
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/hooks/pre-receive.sample
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/hooks/commit-msg.sample
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/hooks/pre-commit.sample
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/hooks/pre-rebase.sample
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/branches
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/.git/modules/third_party/protoc-gen-validate/packed-refs
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/core/ext/upb-generated/envoy/api/v2/core/protocol.upb.c
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/core/ext/upb-generated/envoy/api/v2/core/protocol.upb.h
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/core/tsi/alts/frame_protector/alts_record_protocol_crypter_common.h
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/core/tsi/alts/frame_protector/alts_record_protocol_crypter_common.cc
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/core/tsi/alts/zero_copy_frame_protector/alts_grpc_integrity_only_record_protocol.cc
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/core/tsi/alts/zero_copy_frame_protector/alts_iovec_record_protocol.cc
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/core/tsi/alts/zero_copy_frame_protector/alts_grpc_privacy_integrity_record_protocol.h
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/core/tsi/alts/zero_copy_frame_protector/alts_grpc_privacy_integrity_record_protocol.cc
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/core/tsi/alts/zero_copy_frame_protector/alts_grpc_record_protocol_common.h
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/core/tsi/alts/zero_copy_frame_protector/alts_grpc_record_protocol.h
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/core/tsi/alts/zero_copy_frame_protector/alts_grpc_record_protocol_common.cc
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/core/tsi/alts/zero_copy_frame_protector/alts_iovec_record_protocol.h
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/core/tsi/alts/zero_copy_frame_protector/alts_grpc_integrity_only_record_protocol.h
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/ruby/tools/bin/grpc_tools_ruby_protoc
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/ruby/tools/bin/grpc_tools_ruby_protoc_plugin
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/_split_definitions_test.py
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/_python_plugin_test.py
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/beta_python_plugin_test.py
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/__init__.py
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/requests
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/requests/r
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/requests/r/test_requests.proto
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/requests/r/__init__.py
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/requests/__init__.py
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/responses
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/responses/test_responses.proto
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/responses/__init__.py
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/invocation_testing
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/invocation_testing/split_services
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/invocation_testing/split_services/services.proto
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/invocation_testing/split_messages
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/invocation_testing/split_messages/sub
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/invocation_testing/split_messages/sub/messages.proto
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/invocation_testing/__init__.py
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/invocation_testing/same.proto
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/service
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/service/test_service.proto
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/service/__init__.py
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/__init__.py
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/payload
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/payload/__init__.py
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/src/python/grpcio_tests/tests/protoc_plugin/protos/payload/test_payload.proto

```

And many more entry. 
**Also It's really suspicious that directories are wrote over again and again.** 

For instance:
**/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google**


This is only one directory.

Reply:
I deleted build then tried to run again code but partially it's adding dir into dir:

```
Cloning into '/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/third_party/abseil-cpp'...
Cloning into '/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/third_party/benchmark'...
Cloning into '/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/third_party/bloaty'...
Cloning into '/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/third_party/boringssl'...
Cloning into '/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/third_party/boringssl-with-bazel'...
Cloning into '/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/third_party/cares/cares'...
Cloning into '/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/third_party/envoy-api'...
Cloning into '/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/third_party/gflags'...
Cloning into '/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/third_party/googleapis'...
Cloning into '/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/third_party/googletest'...
Cloning into '/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/third_party/libcxx'...
Cloning into '/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/third_party/libcxxabi'...
Cloning into '/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/third_party/protobuf'...
```

Reply:
> /home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf-build/CMakeFiles/protoc.dir/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/protobuf/protobuf/src/protobuf/src/google

This seems weird, indeed. But the other one:

> Cloning into '/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/grpc/grpc/src/grpc/third_party/abseil-cpp'...

Those seem normal to me (grpc also has a "third_party" directory).

Let's go one step back and try to understand what happens here. First it seems to me that the binaries (protoc and grpc_cpp_plugin) got built successfully, as you have them listed in your `find` commands:

```
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/install/bin/protoc
/home/hanco/Asli/MAVSDK/MAVSDK/build/default/third_party/install/bin/grpc_cpp_plugin
```

The script "simply" tries to find them [at this place](https://github.com/mavlink/MAVSDK/blob/develop/tools/generate_from_protos.sh#L10-L11):

```
third_party_dir="${script_dir}/../build/default/third_party"
protoc_binary="${third_party_dir}/install/bin/protoc"
protoc_grpc_binary="${third_party_dir}/install/bin/grpc_cpp_plugin"
```

And it tells you that it is trying to find them here:

```
/home/hanco/Asli/MAVSDK/MAVSDK/../build/default/third_party
```

For me, that would be:

```
/path/to/mavsdk/tools/../build/default/third_party
```

You see that I have `tools/` in my path, and you don't. But I can see that you ran the script from `./tools/generate_from_protos.sh`, so it should be there.

Could it be that [the following line](https://github.com/mavlink/MAVSDK/blob/develop/tools/generate_from_protos.sh#L5) fails? I.e:

```
script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Let's test that: can you go at [this line](https://github.com/mavlink/MAVSDK/blob/develop/tools/generate_from_protos.sh#L6) in the script, and add:

```
echo ${script_dir}
exit 1
```

Then run the script again and tell me what the output of it is? It should be `/home/hanco/Asli/MAVSDK/MAVSDK/tools/`

Reply:
Yess, you got it. The problem was script directory.

When I added "echo":

`/home/hanco/Asli/MAVSDK/MAVSDK`

And then I just directly add this `script_dir="/home/hanco/Asli/MAVSDK/MAVSDK/tools"` into script to use script without error. Result:

```
/home/hanco/Asli/MAVSDK/MAVSDK/tools
/home/hanco/Asli/MAVSDK/MAVSDK/tools/../build/default/third_party/install/bin/protoc
/home/hanco/Asli/MAVSDK/MAVSDK/tools/../build/default/third_party/install/bin/grpc_cpp_plugin

-------------------------------
 Generating pb and grpc.pb files
    * protoc --version: libprotoc 3.9.1
-------------------------------
```

The problem was that we have to call script in tools directory. For instance:
```

hanco@CRT-boi:~/Asli/MAVSDK/MAVSDK$ ./tools/generate_from_protos.sh 
./tools/generate_from_protos.sh: 5: ./tools/generate_from_protos.sh: Bad substitution
/home/hanco/Asli/MAVSDK/MAVSDK
-------------------------------
 Error
-------------------------------
'protoc' or 'grpc_cpp_plugin' not found

Those are expected to be built for the host system in '/home/hanco/Asli/MAVSDK/MAVSDK/../build/default/third_party'!

You may want to run the CMake configure step first:

    $ cmake -DBUILD_BACKEND=ON -Bbuild/default -S.
hanco@CRT-boi:~/Asli/MAVSDK/MAVSDK$ cd tools/
hanco@CRT-boi:~/Asli/MAVSDK/MAVSDK/tools$ ./generate_from_protos.sh 
./generate_from_protos.sh: 5: ./generate_from_protos.sh: Bad substitution
/home/hanco/Asli/MAVSDK/MAVSDK/tools
/home/hanco/Asli/MAVSDK/MAVSDK/tools/../build/default/third_party/install/bin/protoc
/home/hanco/Asli/MAVSDK/MAVSDK/tools/../build/default/third_party/install/bin/grpc_cpp_plugin

-------------------------------
 Generating pb and grpc.pb files
    * protoc --version: libprotoc 3.9.1
-------------------------------

```

Leaving these aside, should I add now generated files :) If I should, I will follow [this](https://dev.px4.io/v1.9.0/en/contribute/git_examples.html) guide to control which files and lines  are effected by this script. Because I think I cannot find manually.

In this directory `~/Asli/MAVSDK/MAVSDK/src/backend/src/generated/telemetry` and in `telemetry.pb.cc` I can see new modes by automatically:

```
F\020\002\022\010\n\004HOLD\020\003\022\013\n\007MISSION\020\004\022\024\n"
  "\020RETURN_TO_LAUNCH\020\005\022\010\n\004LAND\020\006\022\014\n\010OFFBOAR"
  "D\020\007\022\r\n\tFOLLOW_ME\020\010\022\n\n\006MANUAL\020\t\022\n\n\006ALTCTL"
  "\020\n\022\n\n\006POSCTL\020\013\022\010\n\004ACRO\020\014\022\016\n\nSTABILIZED\020\r"
  "\022\r\n\tRATTITUDE\020\016*\223\001\n\013L
```

Reply:
There is huge differences between previous version.

Reply:
> The problem was that we have to call script in tools directory. 

Hmm that's weird. The point of `script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"` is to make it independent from that :sweat_smile:. What operating system are you running? And do you know which shell?

Reply:
Sorry for lately answer. For future works maybe it can help us:
Operating System:
```
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.3 LTS
Release:	18.04
Codename:	bionic
```
Shell:
```
hanco@CRT-boi:~$ echo $SHELL
/bin/bash
```